RELEASE_HOWTO.md

```diff
@@ -0,0 +1,14 @@
+# Steps for doing a xdg-desktop-portal release
+
+ - git clean -fxd
+ - bump version in configure.ac
+ - add content to NEWS
+ - git commit -m <version>
+ - git push origin master
+ - ./autogen.sh --enable-docbook-docs 
+ - make all
+ - make distcheck
+ - git tag <version>
```

Comment:
@matthiasclasen What do you thinkg of signing the tags with  your GPG key using `gpg -a -s <version>`? No strong feelings, but I think it's a nice thing to do when possible.

Reply:
Feel free to. I don't do gpg

Reply:
Ok, still good to know you're not opposed to that